WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -650,7 +650,7 @@ def execute( self ):
       self.exitWithError( retCode )
 
     # HS06 benchmark
-    cpuNormalizationFactor = float( cpuNormalizationFactorOutput.replace( "Normalization for current CPU is ", '' ).replace( " HS06", '' ) )
+    cpuNormalizationFactor = float( cpuNormalizationFactorOutput.replace( "Estimated CPU power is ", '' ).replace( " HS06", '' ) )
```

Comment:
My apologies for this... I had not realized we were calling the script and parsing the output! Why not call the same method called by the script?
Take care also that if MJF is implemented the script returns two lines: one with that string and one with the MJF value...